test/util/kubernetes.go

```diff
@@ -81,7 +81,7 @@ func GetAppPods(cl kubernetes.Interface, ns string) (map[string][]string, error)
 			return pods, err
 		}
 		items = list.Items
-		ready := true
+		ready := len(items) > 0
 
```

Comment:
This should not be necessary since we now check if the containers are running before declaring the pod ready

Reply:
I also have changes in the same fct in PR, where I check for pods/containers in multiple namespaces. You can revert this.